build/common/config/.golangci.yml

```diff
@@ -68,7 +78,8 @@ linters-settings:
     # report about shadowed variables
     check-shadowing: false
   gci:
-    local-prefixes: open-cluster-management.io/governance-policy-spec-sync
+    sections:
+      - prefix(open-cluster-management.io/config-policy-controller)
```

Comment:
Shoot. I missed this one. Is this supposed to match the repo, or is this intended to be `config-policy-controller`?